### DISCUSSION TITLE
AI: Research and Development

### DISCUSSION THREAD

**Discussion by sondreb** [block-core/ariton#18] [Ideas]
There are many interesting things we plan to do with AI. I have a new Microsoft Surface Copilot+ PC that have a NPU (Neural Processing Unit) that Microsoft is utilizing in many different apps on Windows. We should attempt implement some initial basic SLM (Small Language Models) that can run in the browser, potentially accelerated using the NPU.

One of the first AI features I'd like us to implement is a sentiment analysis in our text editor within Ariton. As user is typing, we could should an emoji based upon analysis of the text a user is writing. This is a very simple form of analysis that have existed for many years.

If anyone is feeling a calling towards working on AI research and development at Ariton, please feel free to learn more and experiment. I can do performance testing with my new Surface device.

**Comment by dangershony** [comment]
It should probably be using some open source model rather then connecting to an existing api right?

**Reply by sondreb** [reply]
Yes, while we might integrate with third party services for different add-on features, this will primarily be running open source models, either on device or services hosted by us.
